### PR TITLE
Fixed broken generic environment completions

### PR DESCRIPTION
**Pull request by dbarashev**
@khazhoyan @yanenkoa review?

### CODE REVIEW

File: lib/ace/ext/papeeria/latex_parsing_context_test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    var Editor = require("ace/editor").Editor;
+    var MockRenderer = require("ace/test/mockrenderer").MockRenderer;
+    var LatexParsingContext = require("ace/ext/papeeria/latex_parsing_context");
+    var PapeeriaLatexHighlightRules = require("ace/ext/papeeria/papeeria_latex_highlight_rules").PapeeriaLatexHighlightRules;
+    Tokenizer = require("ace/tokenizer").Tokenizer;
+    var PapeeriaLatexMode = {
+      getTokenizer: function() {
+        return new Tokenizer((new PapeeriaLatexHighlightRules()).getRules());
+      }
+    };
```
Comment (yanenkoa) [inline lib/ace/ext/papeeria/latex_parsing_context_test.js]: Maybe 4 spaces tabs in this function just to keep it consistent?

File: lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee [hunk 2]
```diff
@@ -25,6 +26,12 @@ define((require, exports, module) ->
   exports.LIST_STATE = LIST_STATE
   exports.EQUATION_TOKENTYPE = EQUATION_TOKENTYPE
   exports.LIST_TOKENTYPE = LIST_TOKENTYPE
+  exports.ENVIRONMENT_STATE = "environment"
```
Comment (yanenkoa) [inline lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee]: @khazhoyan this is not used anywhere, so there's no need for local `ENVIRONMENT_STATE`.

File: lib/ace/ext/papeeria/latex_parsing_context_test.js (hunk 1 above)
Comment (dbarashev) [inline lib/ace/ext/papeeria/latex_parsing_context_test.js]: In fact we normally use 2 spaces for indenting, so I re-indented this file with 2 spaces.

File: lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee (hunk 2 above)
Comment (dbarashev) [inline lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee]: done

File: lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee [hunk 3]
```diff
@@ -11,6 +11,7 @@ define((require, exports, module) ->
   LIST_REGEX = "itemize|enumerate"
   LIST_TOKENTYPE = "list"
 
+  ENVIRONMENT_STATE = ENVIRONMENT_TOKENTYPE = "environment"
```
Comment (dbarashev) [inline lib/ace/ext/papeeria/papeeria_latex_highlight_rules.coffee]: done

File: lib/ace/ext/papeeria/latex_parsing_context_test.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    require("../../test/mockdom");
+}
+
+define(function(require, exports, module) {
+    var EditSession = require("ace/edit_session").EditSession;
+    var Editor = require("ace/editor").Editor;
+    var MockRenderer = require("ace/test/mockrenderer").MockRenderer;
+    var LatexParsingContext = require("ace/ext/papeeria/latex_parsing_context");
+    var PapeeriaLatexHighlightRules = require("ace/ext/papeeria/papeeria_latex_highlight_rules").PapeeriaLatexHighlightRules;
+    Tokenizer = require("ace/tokenizer").Tokenizer;
```
Comment (dbarashev) [inline lib/ace/ext/papeeria/latex_parsing_context_test.js]: fixed, thank

**Pull request merged**